cwl/src/main/scala/cwl/InputParameter.scala

```diff
@@ -0,0 +1,88 @@
+package cwl
+
+import cats.syntax.traverse._
+import common.validation.ErrorOr._
+import common.validation.Validation._
+import shapeless.Poly1
+import wom.types.WomType
+import wom.values.{WomArray, WomValue}
+
+trait InputParameter {
+  def id: String
+  def label: Option[String]
+  def secondaryFiles: Option[SecondaryFiles]
+  def format: Option[Format]
+  def streamable: Option[Boolean]
+  def doc: Option[Doc]
+  def inputBinding: Option[InputCommandLineBinding]
+  def default: Option[CwlAny]
+  def `type`: Option[MyriadInputType]
+
+  def loadContents = inputBinding.flatMap(_.loadContents).getOrElse(false)
+}
+
+object InputParameter {
```

Comment:
Everything (except the unapplies) here is just copied over from `CommandLineTool`